tests/test_invites.py

```diff
@@ -90,6 +90,30 @@ def test_invited_email_address_obfuscation(self):
 
         self.assertEqual(redacted_address, "...@1...")
 
+        # Try using a username separator string
+        self.sydent.third_party_invite_username_separator_string = "-"
+        email_address = "johnathon-jingle-smithington@smiths.notarealtld"
+        redacted_address = store_invite_servlet.redact_email_address(email_address)
+        # Each individual component of the username should be obfuscated, but not the domain
+        self.assertEqual(redacted_address, "johnat...-jin...-smithi...@smiths.n...")
```

Comment:
I think the following makes it clearer that this does *not* apply to the domain:

```suggestion
        email_address = "johnathon-jingle-smithington@john-smith.notarealtld"
        redacted_address = store_invite_servlet.redact_email_address(email_address)
        # Each individual component of the username should be obfuscated, but not the domain
        self.assertEqual(redacted_address, "johnat...-jin...-smithi...@john-smi...")
```